apache2/ag_mdb/ag_mdb_internal.h

```diff
@@ -341,8 +341,8 @@ int Entry_setKeyValue(CPTR_VOID shm_base, PTR_OFFSET entry_id, const char* key,
  ** Define the name prefix of Read/Write lock on Windows
```

Comment:
change comment? probably mentioned that in order to be global namespcace, global should be the prefix

Reply:
Sure, I will change the comment